gravitee-apim-portal-webui-next/src/scss/theme/theme.scss

```diff
@@ -0,0 +1,43 @@
+@use '@angular/material' as mat;
+
+$mat-primary: (
+  main: #6c59bd,
+  lighter: #d3cdeb,
+  darker: #4f3ea6,
+  200: #6c59bd,
+  contrast : (
+    main: #ffffff,
+    lighter: rgba(#000000, 0.87),
+    darker: #ffffff,
+  )
+);
+
+$mat-accent: (
+  main: #e8e3fc,
+  lighter: #f8f7fe,
+  darker: #ded8fb,
+  200: #e8e3fc,
+  contrast : (
+    main: #ffffff,
+    lighter: rgba(#000000, 0.87),
+    darker: #ffffff,
+  )
+);
+
+$mat-warn: (
+  main: #ba1a1a,
+  lighter: #eababa,
+  darker: #a30f0f,
+  200: #ba1a1a,
+  contrast : (
+    main: #ffffff,
+    lighter: rgba(#000000, 0.87),
```

Comment:
Change to `#1d192b` to match Figma designs.

Reply:
Done